config/default

```diff
@@ -10,17 +10,18 @@ import tarfile
 assets = os.path.join(media.mypath, "assets")
 
 if not os.path.exists(assets):
-  print "Extracting assets..."
+  print("Extracting assets...")
   try:
     with tarfile.open("{}.tbz2".format(assets), "r:*") as tf:
       tf.extractall(media.mypath)
-  except tarfile.ReadError, e:
+  except tarfile.ReadError as e:
```

Comment:
Change is in wrong commit.  Please put all "except" fixes into a single commit, together.  The commit should only address "except"... don't combine it with other changes.